docs/source/examples/setup_your_own_data.ipynb

```diff
@@ -0,0 +1,862 @@
+{
```

Comment:
<p>I would get rid of all of these since it's really verbose. You could do <code>help(func)</code>  if one of them is particularly enlightening. Just print out the head of the docstring for each of them for instance. Or explain in the comments what each is used for.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bradyrx/climpred/pull/296/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>